gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -93,8 +90,8 @@ def get_sample_age(sv_list: hl.Table) -> hl.Table:
     ).key_by("s")
 
     # NOTE: Add age to sample list. Most age data is stored as integers in 'age' annotation, # noqa
```

Comment:
```suggestion
    # NOTE: Add age to sample list. Most age data is stored as integers in the 'age' annotation, # noqa
```
(adding "the" for consistency)